contract-base/src/main/kotlin/io/provenance/scope/contract/spec/P8eContract.kt

```diff
@@ -4,18 +4,15 @@ import java.time.OffsetDateTime
 import java.util.UUID
 import java.util.concurrent.atomic.AtomicReference
 
-import io.provenance.scope.contract.proto.Utils
+import io.provenance.scope.proto.Util
 
 /**
  * Provide basic functionality for agreement setup.
  */
 abstract class P8eContract {
-    val uuid = Utils.UUID.newBuilder().setValue(UUID.randomUUID().toString()).build()
+    val uuid = Util.UUID.newBuilder().setValue(UUID.randomUUID().toString()).build()
     val currentTime = AtomicReference<OffsetDateTime?>()
 
-    // By invoking the consideration you are indicating your agreement with the consideration.
-    fun impliedConsent() = Utils.BooleanResult.newBuilder().setValue(true).build()
```

Comment:
I couldn't find this used anywhere, so I think it must be a relic of the past

Reply:
Nice! It is a thing that can get called inside of P8eContract implementations, but I don't see any reason why they can't just set the boolean result directly.